coredb-operator/src/controller.rs

```diff
@@ -209,12 +219,7 @@ impl CoreDB {
                 storage: self.spec.storage.clone(),
                 sharedirStorage: self.spec.sharedirStorage.clone(),
                 pkglibdirStorage: self.spec.pkglibdirStorage.clone(),
-                extensions: self
-                    .status
-                    .as_ref()
-                    .expect("no extensions in `status`")
-                    .extensions
-                    .clone(),
+                extensions: self.status.clone().and_then(|f| f.extensions),
```

Comment:
this was a bug. this shouldn't panic when there are no extensions in status. its an `Option`, and we should allow it to be `Option::None`